src/app/not-found.tsx

```diff
@@ -0,0 +1,117 @@
+import Link from 'next/link'
+import Image from 'next/image'
+import { Text } from '@/components/ui'
+import { MoveLeft } from 'lucide-react'
+import { IconLink } from '@/components/shared/IconLink'
+import { BaseLayout } from '@/components/layout/BaseLayout'
+import DiscordIcon from 'public/assets/vector-icons/discord-icon.svg'
+import TwitterIcon from 'public/assets/vector-icons/twitter-icon.svg'
+import InstagramIcon from 'public/assets/vector-icons/instagram-icon.svg'
+import MailIconInverse from 'public/assets/vector-icons/mail-icon-inverse.svg'
+import { SUPPORT_LINK, DISCORD_LINK, INSTAGRAM_LINK, TWITTER_LINK } from '@/constants/general'
+
+export default function NotFound() {
+  return (
+    <BaseLayout>
+      <div className='flex flex-col sm:w-[660px] justify-center items-center sm:pt-[20px] md:pt-[40px]'>
+        <Text
+          as='h1'
+          styleVariant='primary-heading'
+          className='text-grey-400 uppercase text-center text-7xl leading-[62px] sm:text-9xl sm:leading-[111.36px] tracking-tight'
+        >
+          WE&apos;RE
+          <br />
+          BUILDING
+          <br />
+          SOMETHING
+          <br />
+          GREAT!
+        </Text>
+        <Image
+          src='/assets/images/rabbit-hole.png'
+          alt='dReader Page not found'
+          className='w-[214px] sm:w-[400px] -mt-[47px] sm:-mt-[87px]'
+          width={400}
+          height={1}
+          style={{ height: 'auto' }}
+        />
+        <Text
+          as='p'
+          styleVariant='body-normal'
+          fontWeight='medium'
+          className='text-grey-100 max-sm:hidden text-center leading-snug pt-[38px]'
```

Comment:
no need to use leading here since its defined for style variant and text component

Reply:
Also if its possible to use flex and gap between sections on page instead of adding padding/margin to achieve that